components/pedigree/resources/src/main/resources/pedigree/familyData.js

```diff
@@ -33,8 +33,12 @@ define([
                 this.familyMembersIndex[this.familyMembers[i].id] = i;
             }
 
-            this.warningMessage = familyJSON.hasOwnProperty("warning") ?
-                                  familyJSON.warning : "";
+            this.warningMessage = null;
+            if (familyJSON.hasOwnProperty("contains_sensitive_data") && familyJSON["contains_sensitive_data"]) {
+                this.warningMessage = (familyJSON.hasOwnProperty("sensitive_data_message") && familyJSON["sensitive_data_message"] != "")
+                                      ? familyJSON["sensitive_data_message"]
+                                      : "This pedigree was marked as containing sensitive information. No further details were provided.";
```

Comment:
Can you please make this translatable?

Reply:
there are no (zero) other translations in the pedigree right now. It is a separate issue to introduce translations in pedigree

Reply:
Nevermind, i just noticed it's part of `pedigree` where nothing is translatable.

Reply:
Sounds good, thanks